### PR TITLE
Added info commands

### LINKED ISSUE #871: More info commands
All the points listed below are suggestions. Do as you like

**Text channel info:**
- Channel description
- Creation date
- Display whether the channel is marked as NSFW or not
- Category name
- Channel ID + Mention
- List users/# of users that have access to the channel

**Voice channel info:**
- Bitrate
- Creation date
- Numer of people currently connected
- ID + mention (maybe link to videochat)
- Category name
- User limit

**Emoji info:**
- Creation date
- User that added the emoji
- Display whether it is animated or not
- ID + Raw emoji. Example: "<:emojimester:559517431383588874>"
- Large image of emoji

**Category info:**
- Number of channels in category
- Creation date
- ID

### PR DESCRIPTION
**Pull request by moruzerinho6**
This closes #871

### CODE REVIEW

**Comment by moruzerinho6** [comment]
I need help with the emote command, calling ``emoji.fetchAuthor()`` is making a missing error while I do put a botPermission of ``MANAGE_EMOJIS``

**Comment by pedrofracassi** [comment]
Could you be more specific about the error?

**Comment by moruzerinho6** [comment]
image

**Comment by moruzerinho6** [comment]
I'll add the function inside a try catch since there's no other way.

File: src/commands/utility/info/category.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        required: true,
+        full: true,
+        missingError: 'commands:info.subcommands.voice.missingChannelName'
+      }]
+    }, client)
+  }
+
+  async run ({ message, t, author, channel, language }, givenChannel) {
+    moment.locale(language)
+    givenChannel = message.guild.channels.cache
```
Comment (fernando457829) [inline src/commands/utility/info/category.js:21]: Use `find` and don't reassign a param
```suggestion
    const foundChannel = message.guild.channels.cache.find(channel => channel.type === 'category' && channel.name.toLowerCase() === givenChannel.toLowerCase());
```

File: src/commands/utility/info/category.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      .filter(channel => channel.type === 'category' && channel.name.toLowerCase() === givenChannel.toLowerCase())
+    givenChannel = givenChannel.first()
+    if (!givenChannel) {
+      throw new CommandError(t('errors:channelDoesntExist'))
+    }
+
+    const embed = new SwitchbladeEmbed(author)
+
```
Comment (fernando457829) [inline src/commands/utility/info/category.js:29]: Remove this blank line

File: src/commands/utility/info/channel.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        onlySameGuild: true,
+        missingError: 'commands:move.missingChannelMention'
+      }]
+    }, client)
+  }
+
+  async run ({ t, author, channel, language }, givenChannel) {
+    moment.locale(language)
+    const embed = new SwitchbladeEmbed(author)
+
```
Comment (fernando457829) [inline src/commands/utility/info/channel.js:25]: Remove this blank line.

File: src/commands/utility/info/category.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+const moment = require('moment')
+
+module.exports = class InfoCategory extends Command {
+  constructor (client) {
+    super({
+      name: 'category',
+      parent: 'info',
+      requirements: { guildOnly: true },
+      parameters: [{
+        type: 'string',
```
Comment (fernando457829) [inline src/commands/utility/info/category.js:11]: Will be better if you use `channel` param and customize it to support categories.

File: src/commands/utility/info/emote.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+      }]
+    }, client)
+  }
+
+  async run ({ message, t, author, channel, language }, emoji) {
+    moment.locale(language)
+    try {
+      const emojiAuthor = await emoji.fetchAuthor()
+      const embed = new SwitchbladeEmbed(author)
+
```
Comment (fernando457829) [inline src/commands/utility/info/emote.js:22]: Blank line?

File: src/commands/utility/info/voice.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    moment.locale(language)
+    givenChannel = message.guild.channels.cache
+      .filter(channel => channel.type === 'voice' && channel.name.toLowerCase() === givenChannel.toLowerCase())
+    givenChannel = givenChannel.first()
+    if (!givenChannel) {
+      throw new CommandError(t('errors:channelDoesntExist'))
+    }
+
+    const embed = new SwitchbladeEmbed(author)
+
```
Comment (fernando457829) [inline src/commands/utility/info/voice.js:29]: Blank line?

File: src/commands/utility/info/voice.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+const moment = require('moment')
+
+module.exports = class InfoVoice extends Command {
+  constructor (client) {
+    super({
+      name: 'voice',
+      parent: 'info',
+      requirements: { guildOnly: true },
+      parameters: [{
+        type: 'string',
```
Comment (fernando457829) [inline src/commands/utility/info/voice.js:11]: Will be better if you use channel param and customize it to support voice channels.

File: src/commands/utility/info/voice.js [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+        required: true,
+        full: true,
+        missingError: 'commands:info.subcommands.voice.missingChannelName'
+      }]
+    }, client)
+  }
+
+  async run ({ message, t, author, channel, language }, givenChannel) {
+    moment.locale(language)
+    givenChannel = message.guild.channels.cache
```
Comment (fernando457829) [inline src/commands/utility/info/voice.js:21]: Use `find` and don't reassign a param
```suggestion
    const foundChannel = message.guild.channels.cache.find(channel => channel.type === 'voice' && channel.name.toLowerCase() === givenChannel.toLowerCase());
```

**Review by fernando457829 — DISMISSED** [review]

**Comment by fernando457829** [comment]
LGTM

**Review by pedrofracassi — APPROVED** [review]
LGTM

File: src/locales/en-US/commands.json [hunk 9]
```diff
@@ -1629,6 +1629,43 @@
...
+        "nsfwTag": "[NSFW]"
+      },
+      "voice": {
+        "commandDescription": "Shows a voice channel info.",
+        "commandUsage": "<channelName>",
+        "missingChannelName": "You forgot to give the channel name.",
+        "bitrate": "Bitrate",
+        "userLimit": "User Limit"
+      },
+      "emote": {
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json:1655]:
```suggestion
      "emoji": {
```

File: src/commands/utility/info/emote.js [hunk 10]
```diff
@@ -0,0 +1,35 @@
+const { Command, SwitchbladeEmbed, CommandError } = require('../../../')
+const moment = require('moment')
+
+module.exports = class InfoEmote extends Command {
+  constructor (client) {
+    super({
+      name: 'emote',
```
Comment (pedrofracassi) [inline src/commands/utility/info/emote.js:7]:
```suggestion
      name: 'emoji',
```

File: src/commands/utility/info/emote.js [hunk 11]
```diff
@@ -0,0 +1,35 @@
+const { Command, SwitchbladeEmbed, CommandError } = require('../../../')
+const moment = require('moment')
+
+module.exports = class InfoEmote extends Command {
```
Comment (pedrofracassi) [inline src/commands/utility/info/emote.js:4]:
```suggestion
module.exports = class InfoEmoji extends Command {
```

File: src/locales/en-US/commands.json [hunk 12]
```diff
@@ -1629,6 +1629,43 @@
     "lastUpdated": "Last Updated at",
     "notFound": "Code not found"
   },
+  "info": {
+    "commandDescription": "Shows channels/categories and emojis information.",
+    "channel": "Shows a text/news/announcement channel info.",
+    "embedAuthor": "Discord Information Commands",
+    "voice": "Shows a voice channel info.",
+    "emote": "Shows a emoji info.",
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json:1637]:
```suggestion
    "emoji": "Shows a emoji info.",
```

File: src/locales/en-US/commands.json [hunk 13]
```diff
@@ -1629,6 +1629,43 @@
...
+        "missingChannelName": "You forgot to give the channel name.",
+        "bitrate": "Bitrate",
+        "userLimit": "User Limit"
+      },
+      "emote": {
+        "commandDescription": "Shows the creation date, who added and id of the emoji.",
+        "commandUsage": "<emoji>",
+        "animatedTag": "[Animated]",
+        "whoAdded": "Who added",
+        "missingEmote": "You forgot to send the emoji."
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json:1660]:
```suggestion
        "missingEmoji": "You forgot to send the emoji."
```

File: src/commands/utility/info/emote.js [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+  constructor (client) {
+    super({
+      name: 'emote',
+      parent: 'info',
+      requirements: { guildOnly: true, botPermissions: ['MANAGE_EMOJIS'] },
+      parameters: [{
+        type: 'emoji',
+        missingError: 'commands:info.subcommands.emote.missingEmote'
```
Comment (pedrofracassi) [inline src/commands/utility/info/emote.js:12]:
```suggestion
        missingError: 'commands:info.subcommands.emoji.missingEmoji'
```

File: src/commands/utility/info/emote.js [hunk 15]
```diff
@@ -0,0 +1,35 @@
...
+      }]
+    }, client)
+  }
+
+  async run ({ t, author, channel, language }, emoji) {
+    moment.locale(language)
+    try {
+      const emojiAuthor = await emoji.fetchAuthor()
+      const embed = new SwitchbladeEmbed(author)
+        .setTitle(`${emoji.animated ? `${t('commands:info.subcommands.emote.animatedTag')} ` : ''}${emoji.name}`)
```
Comment (pedrofracassi) [inline src/commands/utility/info/emote.js:22]:
```suggestion
        .setTitle(`${emoji.animated ? `${t('commands:info.subcommands.emoji.animatedTag')} ` : ''}${emoji.name}`)
```

File: src/commands/utility/info/emote.js [hunk 16]
```diff
@@ -0,0 +1,35 @@
...
+        .setDescription(`
+        **${t('commands:guildinfo.createdAt')}**: ${moment(emoji.createdAt).format('LLL')}\n(${moment(emoji.createdAt).fromNow()})
+        **ID**: ${emoji.id} \`\`<:${emoji.name}:${emoji.id}>\`\`
+        ${emojiAuthor ? `**${t('commands:info.subcommands.emote.whoAdded')}**: ${emojiAuthor.tag}` : ''}
```
Comment (pedrofracassi) [inline src/commands/utility/info/emote.js:26]:
```suggestion
        ${emojiAuthor ? `**${t('commands:info.subcommands.emoji.whoAdded')}**: ${emojiAuthor.tag}` : ''}
```

**Review by pedrofracassi — CHANGES_REQUESTED** [review]

File: src/commands/utility/info/category.js [hunk 17]
```diff
@@ -0,0 +1,36 @@
...
+    moment.locale(language)
+    if (!givenChannel) {
+      throw new CommandError(t('errors:channelDoesntExist'))
+    }
+
+    const embed = new SwitchbladeEmbed(author)
+      .setTitle(givenChannel.name)
+      .setDescription(`
+        **${t('commands:guildinfo.createdAt')}**: ${moment(givenChannel.createdAt).format('LLL')}\n(${moment(givenChannel.createdAt).fromNow()})
+        **ID**: ${givenChannel.id}
```
Comment (metehus) [inline src/commands/utility/info/category.js:30]:
```suggestion
        **ID**: \`${givenChannel.id}\`
```

File: src/commands/utility/info/channel.js [hunk 18]
```diff
@@ -0,0 +1,36 @@
...
+      .setTitle(`${givenChannel.nsfw ? `${t('commands:info.subcommands.channel.nsfwTag')} ` : ''}${givenChannel.name}`)
+      .setDescription(
+        `${givenChannel.parent ? `**${t('commands:info.subcommands.channel.category')}**: ${givenChannel.parent.name}` : ''}
+        **${t('commands:guildinfo.createdAt')}**: ${moment(givenChannel.createdAt).format('LLL')}\n(${moment(givenChannel.createdAt).fromNow()})
+        **ID**: ${givenChannel.id} <#${givenChannel.id}>
```
Comment (metehus) [inline src/commands/utility/info/channel.js:30]:
```suggestion
        **ID**: \`${givenChannel.id}\` <#${givenChannel.id}>
```

File: src/commands/utility/info/channel.js [hunk 19]
```diff
@@ -0,0 +1,36 @@
...
+        **${t(givenChannel.members.size > 1 ? t('commands:info.subcommands.channel.canViewThisChannelPlural', { userCount: givenChannel.members.size }) : t('commands:info.subcommands.channel.canViewThisChannel', { userCount: givenChannel.members.size }))}**
```
Comment (metehus) [inline src/commands/utility/info/channel.js:31]: Maybe putting these informations on a separated variable for better reading

File: src/commands/utility/info/emote.js [hunk 20]
```diff
@@ -0,0 +1,35 @@
...
+
+  async run ({ t, author, channel, language }, emoji) {
+    moment.locale(language)
+    try {
+      const emojiAuthor = await emoji.fetchAuthor()
+      const embed = new SwitchbladeEmbed(author)
+        .setTitle(`${emoji.animated ? `${t('commands:info.subcommands.emote.animatedTag')} ` : ''}${emoji.name}`)
+        .setDescription(`
+        **${t('commands:guildinfo.createdAt')}**: ${moment(emoji.createdAt).format('LLL')}\n(${moment(emoji.createdAt).fromNow()})
+        **ID**: ${emoji.id} \`\`<:${emoji.name}:${emoji.id}>\`\`
```
Comment (metehus) [inline src/commands/utility/info/emote.js:25]:
```suggestion
        **ID**: \`${emoji.id}\` \`\`<:${emoji.name}:${emoji.id}>\`\`
```

File: src/commands/utility/info/voice.js [hunk 21]
```diff
@@ -0,0 +1,37 @@
...
+    if (!givenChannel) {
+      throw new CommandError(t('errors:channelDoesntExist'))
+    }
+
+    const embed = new SwitchbladeEmbed(author)
+      .setTitle(givenChannel.name)
+      .setDescription(
+        `${givenChannel.parent ? `**${t('commands:info.subcommands.channel.category')}**: ${givenChannel.parent.name}` : ''}
+        ${t('commands:guildinfo.createdAt')}: ${moment(givenChannel.createdAt).format('LLL')}\n(${moment(givenChannel.createdAt).fromNow()})
+        **ID**: ${givenChannel.id}
```
Comment (metehus) [inline src/commands/utility/info/voice.js:31]:
```suggestion
        **ID**: \`${givenChannel.id}\`
```

File: src/locales/en-US/commands.json [hunk 22]
```diff
@@ -1629,6 +1629,43 @@
...
+        "commandDescription": "Shows channel name, topic, ID, category and how many users can view it.",
+        "commandUsage": "<channel>",
+        "category": "Category",
+        "canViewThisChannel": "{{userCount}} User can view this channel.",
+        "canViewThisChannelPlural": "{{userCount}} Users can view this channel.",
+        "nsfwTag": "[NSFW]"
+      },
+      "voice": {
+        "commandDescription": "Shows a voice channel info.",
+        "commandUsage": "<channelName>",
```
Comment (metehus) [inline src/locales/en-US/commands.json:1650]:
```suggestion
        "commandUsage": "<channel name>",
```

File: src/locales/en-US/commands.json [hunk 23]
```diff
@@ -1629,6 +1629,43 @@
     "lastUpdated": "Last Updated at",
     "notFound": "Code not found"
   },
+  "info": {
+    "commandDescription": "Shows channels/categories and emojis information.",
+    "channel": "Shows a text/news/announcement channel info.",
+    "embedAuthor": "Discord Information Commands",
+    "voice": "Shows a voice channel info.",
+    "emote": "Shows a emoji info.",
+    "category": "Shows a category info.",
```
Comment (metehus) [inline src/locales/en-US/commands.json:1638]:
```suggestion
    "category": "Shows a category channel info.",
```

**Review by metehus — CHANGES_REQUESTED** [review]

File: src/commands/utility/info/channel.js (hunk 19 above)
Comment (moruzerinho6) [reply src/commands/utility/info/channel.js:31]: Example

Comment (metehus) [reply src/commands/utility/info/channel.js:31]: Well, take the last line for example:
```js
        **${t(givenChannel.members.size > 1 ? t('commands:info.subcommands.channel.canViewThisChannelPlural', { userCount: givenChannel.members.size }) : t('commands:info.subcommands.channel.canViewThisChannel', { userCount: givenChannel.members.size }))}**
```

this line is very big, you could turn this into a variable, like:

```js
	const canViewText = t(`commands:info.subcommands.channel.canViewThisChannel${givenChannel.members.size > 1 ? 'Plural : ''}`, userCount: givenChannel.members.size)
	...
		**${canView}**
```

btw, why is there a `t` inside another `t`?
```
${t(givenChannel.members.size > 1 ? t('...', { ... }) : ...
  ^                                 ^ 
```

**Review by pedrofracassi — APPROVED** [review]

File: src/commands/utility/info/category.js [hunk 24]
```diff
@@ -0,0 +1,36 @@
...
+    super({
+      name: 'category',
+      parent: 'info',
+      requirements: { guildOnly: true },
+      parameters: [{
+        type: 'channel',
+        acceptCategory: true,
+        required: true,
+        full: true,
+        missingError: 'commands:info.subcommands.voice.missingChannelName'
```
Comment (jaxxibae) [inline src/commands/utility/info/category.js:15]: Wrong string. Should be `That category doesn't exist or I couldn't find it.`

File: src/commands/utility/info/category.js [hunk 25]
```diff
@@ -0,0 +1,36 @@
...
+
+module.exports = class InfoCategory extends Command {
+  constructor (client) {
+    super({
+      name: 'category',
+      parent: 'info',
+      requirements: { guildOnly: true },
+      parameters: [{
+        type: 'channel',
+        acceptCategory: true,
```
Comment (jaxxibae) [inline src/commands/utility/info/category.js:12]: It doesn't accept category IDs, like 445631099440463872 for example

File: src/commands/utility/info/emote.js [hunk 26]
```diff
@@ -0,0 +1,35 @@
...
+const moment = require('moment')
+
+module.exports = class InfoEmoji extends Command {
+  constructor (client) {
+    super({
+      name: 'emoji',
+      parent: 'info',
+      requirements: { guildOnly: true, botPermissions: ['MANAGE_EMOJIS'] },
+      parameters: [{
+        type: 'emoji',
```
Comment (jaxxibae) [inline src/commands/utility/info/emote.js:11]: It doesn't accept emoji IDs, like 445631099440463872 for example

File: src/commands/utility/info/emote.js [hunk 27]
```diff
@@ -0,0 +1,35 @@
...
+
+  async run ({ t, author, channel, language }, emoji) {
+    moment.locale(language)
+    try {
+      const emojiAuthor = await emoji.fetchAuthor()
+      const embed = new SwitchbladeEmbed(author)
+        .setTitle(`${emoji.animated ? `${t('commands:info.subcommands.emoji.animatedTag')} ` : ''}${emoji.name}`)
+        .setDescription(`
+        **${t('commands:guildinfo.createdAt')}**: ${moment(emoji.createdAt).format('LLL')}\n(${moment(emoji.createdAt).fromNow()})
+        **ID**: \`${emoji.id}\` \`\`<:${emoji.name}:${emoji.id}>\`\`
```
Comment (jaxxibae) [inline src/commands/utility/info/emote.js:25]:
```suggestion
        **ID**: \`${emoji.id}\` \`\`${emoji.toString()}\`\`
```
Animated emojis don't display the tag correctly. and by using `emoji.toString()`, it'll return the right format for that emoji, regardless of it being animated or not.

File: src/commands/utility/info/voice.js [hunk 28]
```diff
@@ -0,0 +1,37 @@
...
+      throw new CommandError(t('errors:channelDoesntExist'))
+    }
+
+    const embed = new SwitchbladeEmbed(author)
+      .setTitle(givenChannel.name)
+      .setDescription(
+        `${givenChannel.parent ? `**${t('commands:info.subcommands.channel.category')}**: ${givenChannel.parent.name}` : ''}
+        ${t('commands:guildinfo.createdAt')}: ${moment(givenChannel.createdAt).format('LLL')}\n(${moment(givenChannel.createdAt).fromNow()})
+        **ID**: \`${givenChannel.id}\`
+        **${t('commands:info.subcommands.voice.bitrate')}**: ${givenChannel.bitrate}
```
Comment (jaxxibae) [inline src/commands/utility/info/voice.js:32]:
```suggestion
        **${t('commands:info.subcommands.voice.bitrate')}**: ${givenChannel.bitrate / 1000}kbps
```
image

File: src/locales/en-US/commands.json [hunk 29]
```diff
@@ -1629,6 +1629,43 @@
...
+  "info": {
+    "commandDescription": "Shows channels/categories and emojis information.",
+    "channel": "Shows a text/news/announcement channel info.",
+    "embedAuthor": "Discord Information Commands",
+    "voice": "Shows a voice channel info.",
+    "emoji": "Shows a emoji info.",
+    "category": "Shows a category channel info.",
+    "subcommands": {
+      "channel": {
+        "commandDescription": "Shows channel name, topic, ID, category and how many users can view it.",
```
Comment (jaxxibae) [inline src/locales/en-US/commands.json:1641]:
```suggestion
        "commandDescription": "Shows a channel's name, topic, ID, category and how many users can view it.",
```

File: src/locales/en-US/commands.json [hunk 30]
```diff
@@ -1629,6 +1629,43 @@
     "lastUpdated": "Last Updated at",
     "notFound": "Code not found"
   },
+  "info": {
+    "commandDescription": "Shows channels/categories and emojis information.",
+    "channel": "Shows a text/news/announcement channel info.",
+    "embedAuthor": "Discord Information Commands",
+    "voice": "Shows a voice channel info.",
```
Comment (jaxxibae) [inline src/locales/en-US/commands.json:1636]:
```suggestion
    "voice": "Shows information about a voice channel.",
```

File: src/locales/en-US/commands.json [hunk 31]
```diff
@@ -1629,6 +1629,43 @@
...
+    "emoji": "Shows a emoji info.",
```
Comment (jaxxibae) [inline src/locales/en-US/commands.json:1637]:
```suggestion
    "emoji": "Shows information about an emoji.",
```

File: src/locales/en-US/commands.json [hunk 32]
```diff
@@ -1629,6 +1629,43 @@
     "lastUpdated": "Last Updated at",
     "notFound": "Code not found"
   },
+  "info": {
+    "commandDescription": "Shows channels/categories and emojis information.",
+    "channel": "Shows a text/news/announcement channel info.",
```
Comment (jaxxibae) [inline src/locales/en-US/commands.json:1634]:
```suggestion
    "channel": "Shows information about a text/news/announcement channel.",
```

File: src/locales/en-US/commands.json [hunk 33]
```diff
@@ -1629,6 +1629,43 @@
...
+    "embedAuthor": "Discord Information Commands",
+    "voice": "Shows a voice channel info.",
+    "emoji": "Shows a emoji info.",
+    "category": "Shows a category channel info.",
```
Comment (jaxxibae) [inline src/locales/en-US/commands.json:1638]:
```suggestion
    "category": "Shows information about a category.",
```

File: src/locales/en-US/commands.json [hunk 34]
```diff
@@ -1629,6 +1629,43 @@
...
+        "commandUsage": "<channel>",
+        "category": "Category",
+        "canViewThisChannel": "{{userCount}} User can view this channel.",
+        "canViewThisChannelPlural": "{{userCount}} Users can view this channel.",
+        "nsfwTag": "[NSFW]"
+      },
+      "voice": {
+        "commandDescription": "Shows a voice channel info.",
+        "commandUsage": "<channel name>",
+        "missingChannelName": "You forgot to give the channel name.",
```
Comment (jaxxibae) [inline src/locales/en-US/commands.json:1651]:
```suggestion
        "missingChannelName": "You need to give me the channel name to get information from.",
```

File: src/locales/en-US/commands.json [hunk 35]
```diff
@@ -1629,6 +1629,43 @@
...
+        "bitrate": "Bitrate",
+        "userLimit": "User Limit"
+      },
+      "emoji": {
+        "commandDescription": "Shows the creation date, who added and id of the emoji.",
```
Comment (jaxxibae) [inline src/locales/en-US/commands.json:1656]:
```suggestion
        "commandDescription": "Shows information about an emoji, such as its creation date, who added it, and its ID.",
```

File: src/locales/en-US/commands.json [hunk 36]
```diff
@@ -1629,6 +1629,43 @@
...
+      "channel": {
+        "commandDescription": "Shows channel name, topic, ID, category and how many users can view it.",
+        "commandUsage": "<channel>",
+        "category": "Category",
+        "canViewThisChannel": "{{userCount}} User can view this channel.",
+        "canViewThisChannelPlural": "{{userCount}} Users can view this channel.",
+        "nsfwTag": "[NSFW]"
+      },
+      "voice": {
+        "commandDescription": "Shows a voice channel info.",
```
Comment (jaxxibae) [inline src/locales/en-US/commands.json:1649]:
```suggestion
        "commandDescription": "Shows information about a voice channel.",
```

File: src/locales/en-US/commands.json [hunk 37]
```diff
@@ -1629,6 +1629,43 @@
...
+      },
+      "emoji": {
+        "commandDescription": "Shows the creation date, who added and id of the emoji.",
+        "commandUsage": "<emoji>",
+        "animatedTag": "[Animated]",
+        "whoAdded": "Who added",
+        "missingEmoji": "You forgot to send the emoji."
+      },
+      "category": {
+        "commandDescription": "Shows category ID, creation date and how many channels are part of it.",
```
Comment (jaxxibae) [inline src/locales/en-US/commands.json:1663]:
```suggestion
        "commandDescription": "Shows a category's ID, creation date and how many channels are part of it.",
```

File: src/commands/utility/info/emote.js [hunk 38]
```diff
@@ -0,0 +1,35 @@
...
+        ${emojiAuthor ? `**${t('commands:info.subcommands.emoji.whoAdded')}**: ${emojiAuthor.tag}` : ''}
+        `)
+        .setImage(emoji.url)
+
+      channel.send(embed)
+    } catch (e) {
+      throw new CommandError(t('errors:needManageEmojis'))
```
Comment (jaxxibae) [inline src/commands/utility/info/emote.js:32]: Why is this needed?

**Review by jaxxibae — DISMISSED** [review]

File: src/commands/utility/info/category.js (hunk 24 above)
Comment (moruzerinho6) [reply src/commands/utility/info/category.js:15]: Category is a channel

File: src/commands/utility/info/category.js (hunk 25 above)
Comment (moruzerinho6) [reply src/commands/utility/info/category.js:12]: Yeah, because the channel parameter never did.

File: src/commands/utility/info/emote.js (hunk 26 above)
Comment (moruzerinho6) [reply src/commands/utility/info/emote.js:11]: Yeah, because the emoji parameter never did.

File: src/commands/utility/info/emote.js (hunk 38 above)
Comment (moruzerinho6) [reply src/commands/utility/info/emote.js:32]: > I need help with the emote command, calling `emoji.fetchAuthor()` is making a missing error while I do put a botPermission of `MANAGE_EMOJIS`

> I'll add the function inside a try catch since there's no other way.

Comment (jaxxibae) [reply src/commands/utility/info/emote.js:32]: What I believe is happening is that the bot is ignoring command permissions, I'll look into it a bit more

Comment (jaxxibae) [reply src/commands/utility/info/emote.js:32]: I've understood the error. It's trying to verify the permissions on the channel that the message is being sent, rather than verifying it on the guild itself. I'll try to make a fix locally, and I'll push to this branch

Comment (jaxxibae) [reply src/commands/utility/info/emote.js:32]: Oh, I see. It's trying to verify if the bot has permissions, but opts.permissions doesn't exists, that's why it's returning true every time. Blame @davipatury.
image

Comment (jaxxibae) [reply src/commands/utility/info/emote.js:32]: Yep, now working as intended. 
image

File: src/commands/utility/info/category.js (hunk 24 above)
Comment (jaxxibae) [reply src/commands/utility/info/category.js:15]: Yeah, I know, but it's just to look better, yk

File: src/commands/utility/info/category.js (hunk 25 above)
Comment (jaxxibae) [reply src/commands/utility/info/category.js:12]: Oh

File: src/commands/utility/info/emote.js (hunk 26 above)
Comment (jaxxibae) [reply src/commands/utility/info/emote.js:11]: Oh^2

File: src/commands/utility/info/channel.js (hunk 3 above)
Comment (almeidx) [reply src/commands/utility/info/channel.js:25]: it's still here

File: src/locales/en-US/errors.json [hunk 39]
```diff
@@ -48,6 +48,9 @@
   "validLinkButGhostMessage": "Your link is valid but I couldn't find that message.",
   "invalidMessageLink": "The message link you sent is invalid.",
   "messageContainsNothing": "The message you're trying to move has no content.",
-  "guildHasNoEmoji": "This server has no emoji or all emojis are not available for me.",
-  "guildHasNoRole": "This server has no role to list."
+  "forgotCheckType": "You forgot to give what I'm checking.",
+  "forgotCheckValue": "You forgot to give the value of what I'm checking.",
+  "channelDoesntExist": "That channel doesn't exist or I couldn't find it.",
+  "guildHasNoRole": "This server has no roles to list.",
+  "guildHasNoEmoji": "This server has no emoji or no emojis are available for me."
```
Comment (almeidx) [inline src/locales/en-US/errors.json:55]: This should be enough
```suggestion
  "guildHasNoEmoji": "This server has no available emojis for me."
```

**Review by almeidx — CHANGES_REQUESTED** [review]

**Review by almeidx — APPROVED** [review]

**Review by pedrofracassi — COMMENTED** [review]
LGTM

**Review by pedrofracassi — APPROVED** [review]
